go/store/blobstore/local.go

```diff
@@ -232,3 +211,22 @@ func (bs *LocalBlobstore) Exists(ctx context.Context, key string) (bool, error)
 
 	return err == nil, err
 }
+
+func (bs *LocalBlobstore) Contatenate(ctx context.Context, key string, sources []string) (ver string, err error) {
+	readers := make([]io.Reader, len(sources))
+	for i := range readers {
+		path := filepath.Join(bs.RootDir, sources[i]) + bsExt
+		if readers[i], err = os.Open(path); err != nil {
+			return "", err
+		}
+	}
+
+	ver, err = bs.Put(ctx, key, io.MultiReader(readers...))
+
+	for i := range readers {
+		if cerr := readers[i].(io.Closer).Close(); err != nil {
```

Comment:
Nit: Should Reader above be ReadCloser?

Reply:
initially it was, but `io.MultiReader()` can only take a `[]io.Reader` not an `[]io.ReadCloser` (furp)